workspaces/analytics/plugins/analytics-module-ga/dev/index.tsx

```diff
@@ -13,15 +13,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import React from 'react';
-import { createDevApp } from '@backstage/dev-utils';
-import { Playground } from './Playground';
 import {
   analyticsApiRef,
   configApiRef,
   identityApiRef,
 } from '@backstage/core-plugin-api';
+import { createDevApp } from '@backstage/dev-utils';
+import React from 'react';
 import { GoogleAnalytics } from '../src';
+import { Playground } from './Playground';
```

Comment:
I saw that in different PRs, can we stay with the old order? I know, its an automated change, but this diff looks odd to me.

I didn't checked yet which config is responsible. Ping me if I should take a deeper look why some workspaces or plugins do this change and others not.

Wdyt @awanlin @04kash ?